pkgs/desktops/xfce/core/thunar/default.nix

```diff
@@ -44,9 +44,9 @@ let unwrapped = mkXfceDerivation {
     xfconf
   ];
 
-  patches = [
-    ./thunarx_plugins_directory.patch
```

Comment:
Can we remove this [patch](https://github.com/NixOS/nixpkgs/blob/d46fa9eb57272b1eb58646ea5075c45203a18c7f/pkgs/desktops/xfce/core/thunar/thunarx_plugins_directory.patch) now?

Reply:
I thought i deleted it, Thanks!